src/neo/Utility.cs

```diff
@@ -6,6 +8,28 @@ namespace Neo
 {
     public static class Utility
     {
+        internal class Logger : ReceiveActor
+        {
+            public Logger()
+            {
+                Receive<Debug>(e => OnLog(LogLevel.Debug, e.ToString()));
+                Receive<Info>(e => OnLog(LogLevel.Info, e.ToString()));
+                Receive<Warning>(e => OnLog(LogLevel.Warning, e.ToString()));
+                Receive<Error>(e => OnLog(LogLevel.Error, e.ToString()));
+                Receive<InitializeLogger>(_ => Init(Sender));
```

Comment:
No `Fatal`?

Reply:
Akka can't find `Fatal`, I think that akka only have error, warning, info, and debug